components/login/passwordReset/src/domain/password/Repositories/HTTPPasswordRepository.js

```diff
@@ -3,10 +3,11 @@ import {inlineError} from '@s-ui/decorators'
 import {PasswordRepository} from './PasswordRepository.js'
 
 export class HTTPPasswordRepository extends PasswordRepository {
-  constructor({config, fetcher}) {
+  constructor({config, fetcher, passwordError}) {
     super()
     this._config = config
     this._fetcher = fetcher
+    this._passwordError = passwordError
```

Comment:
what do you think about rename it this way? ⬇️ 
```suggestion
    this._passwordErrorFactory = passwordErrorFactory
```

Reply:
I don't agree with the proposal 😋 
IMHO including the factory word in the property name may induce to  think that the property contains a reference to the fully factory class. And it only contains the specific `passwordError` initializer.